src/dmqtest/util/Record.d

```diff
@@ -18,24 +18,36 @@ module dmqtest.util.Record;
 
 *******************************************************************************/
 
-import ocean.text.convert.Formatter;
-
 import ocean.transition;
 
 
 /*******************************************************************************
 
     Generates a record value deterministically from the provided integer.
 
+    Leading zeroes are added if `i` has less decimal digits than `digits`. If
+    `i` has more decimal digits then the higher digits are truncated.
+
     Params:
         i = integer to generate record from
+        digits = the number of digits to use
 
     Returns:
         record value corresponding to i
 
 *******************************************************************************/
 
-istring getRecord ( uint i )
+istring getRecord ( uint i, uint digits )
```

Comment:
```D
auto fmt_str = format("{{:d{}}", digits);
return format(fmt_str, i);
```

Reply:
To be honest, you could probably just make `digits` a compile-time constant (16?). There's no real reason to calculate the exact minimum number of digits required, is there?

Reply:
Yeah, good idea.